src/lib/compiler/eventTypes.js

```diff
@@ -71,6 +72,7 @@ export const EventNames = {
   [EVENT_IF_VALUE]: "If Variable Has Value",
   [EVENT_IF_INPUT]: "If Joypad Input Pressed",
   [EVENT_IF_ACTOR_AT_POSITION]: "If Actor At Position",
+  [EVENT_IF_ACTOR_DIRECTION]: "If Actor Direction matches",
```

Comment:
Could you please change this to "If Actor Facing Direction"

Reply:
Done 🙂